example/spec/standalone/redeployment/WorkflowRedeploySpec.js

```diff
@@ -1,128 +0,0 @@
-'use strict';
-
-const pRetry = require('p-retry');
-
-const {
-  buildAndStartWorkflow,
-  waitForCompletedExecution,
-  executionsApi: executionsApiTestUtils
-} = require('@cumulus/integration-tests');
-
-const {
-  loadConfig,
-  protectFile,
-  runKes
-} = require('../../helpers/testUtils');
-
-const {
-  removeWorkflow,
-  removeTaskFromWorkflow
-} = require('../../helpers/workflowUtils');
-
-const workflowsYmlFile = './workflows.yml';
-
-const timeout = 30 * 60 * 1000; // Timout for test setup/teardown in milliseconds
-const deployTimeout = 15; // deployment timeout in minutes
-
-function redeployWithRetries(config) {
-  return pRetry(
-    () => runKes(config, { timeout: deployTimeout }),
-    {
-      retries: 2
-    }
-  );
-}
-
-describe('When a workflow', () => {
```

Comment:
I would prefer `xdescribe`ing this test, with a `TODO: reimplement for Terraform` note.